runtimes/pythonrt/runner_manager.go

```diff
@@ -51,35 +49,20 @@ type Healthier interface {
 	Health(ctx context.Context, in *userCode.RunnerHealthRequest, opts ...grpc.CallOption) (*userCode.RunnerHealthResponse, error)
 }
 
-func waitForServer(ctx context.Context, name string, h Healthier, timeout time.Duration) error {
-	ctx, cancel := context.WithTimeout(ctx, timeout)
-	defer cancel()
-	start := time.Now()
-	var req userCode.RunnerHealthRequest
-
-	for time.Since(start) <= timeout {
-		resp, err := h.Health(ctx, &req)
-		if err != nil || resp.Error != "" {
-			time.Sleep(10 * time.Millisecond)
-			continue
-		}
-
-		return nil
-	}
-
-	return fmt.Errorf("%s not ready after %v", name, timeout)
-}
-
 func dialRunner(ctx context.Context, addr string) (*RunnerClient, error) {
 	ctx, span := telemetry.T().Start(ctx, "dialRunner")
 	defer span.End()
 
 	span.SetAttributes(attribute.String("addr", addr))
 
 	creds := insecure.NewCredentials()
+	params := grpc.ConnectParams{
```

Comment:
please add comments here explainig why this is the way it is and how it waits to the connection to be open etc